test/files/stream.iced

```diff
@@ -0,0 +1,52 @@
+crypto = require('crypto')
+to_buf = require('../../src/stream-to-buffer.iced')
+stream = require('../../src/chunk-stream.iced')
+
+stream_random_data = (strm, len, cb) ->
+  written = 0
+  expected_results = []
+  while written < len
+    # generate random length
+    await crypto.randomBytes(1, defer(err, index))
```

Comment:
actually wait, we don't need to do this - if `T.equal(expected, stb.getBuffer())` fails, it will log the input and output buffers, which you can use to recreate the test.